deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Common Labels
+labels:
+  app: "adapter-master-data-service"
+  group: "rainmaker"
```

Comment:
This isn't rainmaker project. Name it `ifix`.

Reply:
Changed